deploy/helm/elastic-agent/templates/agent/cluster-role.yaml

```diff
@@ -0,0 +1,91 @@
+{{- include "elasticagent.init" $ -}}
+{{- range $presetName, $presetVal := $.Values.agent.presets -}}
+{{/* cluster role binding is not namespace bound so let's try to give it a unique enough name */}}
+{{- $clusterRoleName := printf "agent-%s" (cat $presetName $.Release.Name $.Release.Namespace | sha1sum) -}}
```

Comment:
I would prefer this to be more informative. The name should include the Helm Chart name, the release name and the namespace in plain text, imo. The full SHA-1 is also a bit too long.

Reply:
above comment